### DISCUSSION TITLE
Shortcuts

### DISCUSSION THREAD

**Discussion by lwbaqueros** [Lumieducation/Lumi#1582] [Q&A]
**Where can I find Lumi shortcuts?**

For example, I am looking for a shortcut to switch between the `VIEW` and `EDIT` tabs. 

It really improves my workflow, as every time I want to see a change in the content I am creating, I have to scroll up with my mouse and then click on the `View` tab, and some contents are long: I spent too much time just to see how my content looks like.

**Comment by sr258** [comment]
There currently are no shortcuts besides those in the main menu. We'll add this to the feature request list, though.
